feat: support api v10 (#353)
DisnakeDev/disnake: docs/api.rst

        checking the user IDs. Note that :class:`~ext.commands.Bot` does not
        have this problem.

    .. note::

>         Not all messages may have content. This is a discord limitation.

Reviewer: ```suggestion
        Not all messages will have the `content` parameter. This is a discord limitation.
```

    If this occurs increase the :class:`max_messages <Client>` parameter
    or use the :func:`on_raw_message_edit` event instead.

    .. note::

>         Not all messages may have content. This is a discord limitation.

Reviewer: ```suggestion
        Not all messages will have the `content` parameter. This is a discord limitation.
```

Reviewer 2: I agree with @abhigyantrips regarding the capitalization of "discord" here, but I feel like changing this to "content parameter" would be inaccurate since the intent applies to other fields as well (namely components, attachments, and embeds).
